lib/ace/ext/papeeria/snippets/EquationSnippets.coffee

```diff
@@ -114,14 +100,14 @@ define((require, exports, module) ->
     "\\varkappa"
   ])
 
-  greekLetters = greekLetters.map((word) ->
+  GREEK_LETTERS = GREEK_LETTERS.map((word) ->
     caption: word,
     value: word
-    score: 2
+    score: 1000
```

Comment:
why don't you assign different score to different letters, in lexicographical order?